finagle-core/src/main/scala/com/twitter/finagle/Path.scala

```diff
@@ -103,6 +84,27 @@ object Path {
     true
   }
 
+  // We're extra careful with allocation here because any time
+  // there are nonbase delegations, we need to serialize the paths
+  // to strings
+  private[finagle] def showElem(buf: Buf): String = {
```

Comment:
can we make this a val'd up function?  as it is, we're entailing an allocation when we placeholder syntax the method right now.